Error loading configuration: [Errno 2] No such file or directory: 'config/config.yaml'
✓ Konfigurasi loaded: {'project_settings': {'default_crs': 'EPSG:4326'}}
Mencoba membaca file: D:\PROJECT\Simulasi-Kasus-Lahan\data\raw\sample_land_data.shp
Successfully read shapefile: D:\PROJECT\Simulasi-Kasus-Lahan\data\raw\sample_land_data.shp
✓ Data loaded dengan 100 records
Cleaned data: 100 records
✓ Data berhasil dibersihkan
Stored 'gdf_clean' (GeoDataFrame)


ANALISIS POLA SPASIAL
==================================================
C:\Users\ThinkPad\AppData\Local\Temp\ipykernel_9988\1872476208.py:6: UserWarning:

Geometry is in a geographic CRS. Results from 'centroid' are likely incorrect. Use 'GeoSeries.to_crs()' to re-project geometries to a projected CRS before this operation.


C:\Users\ThinkPad\AppData\Local\Temp\ipykernel_9988\1872476208.py:7: UserWarning:

Geometry is in a geographic CRS. Results from 'centroid' are likely incorrect. Use 'GeoSeries.to_crs()' to re-project geometries to a projected CRS before this operation.


C:\Users\ThinkPad\AppData\Local\Temp\ipykernel_9988\1872476208.py:8: UserWarning:

Geometry is in a geographic CRS. Results from 'centroid' are likely incorrect. Use 'GeoSeries.to_crs()' to re-project geometries to a projected CRS before this operation.




C:\Users\ThinkPad\AppData\Local\Temp\ipykernel_9988\3745205743.py:6: UserWarning:

Column names longer than 10 characters will be truncated when saved to ESRI Shapefile.

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[9], line 6
      3 os.makedirs(output_dir, exist_ok=True)
      5 # Simpan data yang sudah dibersihkan
----> 6 gdf_clean.to_file(f'{output_dir}/cleaned_land_data.shp')
      7 print(f"Data cleaned disimpan ke: {output_dir}/cleaned_land_data.shp")
      9 # Simpan statistik deskriptif

File ~\AppData\Local\Programs\Python\Python313\Lib\site-packages\geopandas\geodataframe.py:1637, in GeoDataFrame.to_file(self, filename, driver, schema, index, **kwargs)
   1543 """Write the ``GeoDataFrame`` to a file.
   1544 
   1545 By default, an ESRI shapefile is written, but any OGR data source
   (...)   1633 
   1634 """
   1635 from geopandas.io.file import _to_file
-> 1637 _to_file(self, filename, driver, schema, index, **kwargs)

File ~\AppData\Local\Programs\Python\Python313\Lib\site-packages\geopandas\io\file.py:718, in _to_file(df, filename, driver, schema, index, mode, crs, engine, metadata, **kwargs)
    711     warnings.warn(
    712         "Column names longer than 10 characters will be truncated when saved to "
    713         "ESRI Shapefile.",
    714         stacklevel=3,
    715     )
    717 if (df.dtypes == "geometry").sum() > 1:
--> 718     raise ValueError(
    719         "GeoDataFrame contains multiple geometry columns but GeoDataFrame.to_file "
    720         "supports only a single geometry column. Use a GeoDataFrame.to_parquet or "
    721         "GeoDataFrame.to_feather, drop additional geometry columns or convert them "
    722         "to a supported format like a well-known text (WKT) using "
    723         "`GeoSeries.to_wkt()`.",
    724     )
    725 _check_metadata_supported(metadata, engine, driver)
    727 if mode not in ("w", "a"):

ValueError: GeoDataFrame contains multiple geometry columns but GeoDataFrame.to_file supports only a single geometry column. Use a GeoDataFrame.to_parquet or GeoDataFrame.to_feather, drop additional geometry columns or convert them to a supported format like a well-known text (WKT) using `GeoSeries.to_wkt()`.